src/styles/settings/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__stars &__star:nth-child(1n) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__stars &__star:nth-child(1n + 5) {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
The selector `&__stars &__star:nth-child(1n + 5)` is intended to target the fifth star and beyond. Ensure this is the desired behavior, as it might not be clear without context.